src/main/java/mate/academy/springboot/swagger/service/impl/ProductServiceImpl.java

```diff
@@ -0,0 +1,50 @@
+package mate.academy.springboot.swagger.service.impl;
+
+import java.math.BigDecimal;
+import java.util.List;
+import java.util.Optional;
+import mate.academy.springboot.swagger.model.Product;
+import mate.academy.springboot.swagger.repository.ProductRepository;
+import mate.academy.springboot.swagger.service.ProductService;
+import org.springframework.data.domain.PageRequest;
+import org.springframework.stereotype.Service;
+
+@Service
+public class ProductServiceImpl implements ProductService {
+    private final ProductRepository productRepository;
+
+    public ProductServiceImpl(ProductRepository productRepository) {
+        this.productRepository = productRepository;
+    }
+
+    @Override
+    public Product create(Product product) {
+        return productRepository.save(product);
+    }
+
+    @Override
+    public Optional<Product> get(Long id) {
```

Comment:
Let's method return an entity instead of Optional and throw NoSuchElementException if entity was not found.